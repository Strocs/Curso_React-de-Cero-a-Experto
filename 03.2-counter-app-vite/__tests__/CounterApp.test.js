import { expect, test, describe } from "vitest"

describe("Pruebas en <CounterApp />", () => {
	test("should not fail", () => {
		const message = "Hello World"
		const message2 = message.trim()

		expect(message).toBe(message2)
	})
})
