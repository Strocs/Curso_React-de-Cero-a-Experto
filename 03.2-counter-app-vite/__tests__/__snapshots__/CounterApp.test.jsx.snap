// Vitest Snapshot v1

exports[`Pruebas en <CounterApp /> > debe de hacer match con el snapshot 1`] = `
<div>
  <h1>
    Counter App
  </h1>
  <h2>
    10
  </h2>
  <div>
    <button
      type="button"
    >
      +1
    </button>
    <button
      type="button"
    >
      -1
    </button>
    <button
      aria-label="btn-reset"
      type="button"
    >
      Reset
    </button>
  </div>
</div>
`;
